functions/__sf_section_host.fish

```diff
@@ -0,0 +1,52 @@
+#
+# Hostname
+#
+
+
+# If there is an ssh connections, current machine name.
+function __sf_section_host -d "Display the current hostname if connected over SSH"
+
+  # ------------------------------------------------------------------------------
+  # Configuration
+  # ------------------------------------------------------------------------------
+
+  # --------------------------------------------------------------------------
+  # | SPACEFISH_HOST_SHOW | show hostname on local | show hostname on remote |
+  # |---------------------+------------------------+-------------------------|
+  # | false               | false                  | false                   |
+  # | always              | true                   | true                    |
+  # | true                | true                   | false                   |
+  # --------------------------------------------------------------------------
+
+  __sf_util_set_default SPACEFISH_HOST_SHOW true
+  __sf_util_set_default SPACEFISH_HOST_PREFIX "at "
+  __sf_util_set_default SPACEFISH_HOST_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+  __sf_util_set_default SPACEFISH_HOST_COLOR "blue"
+  __sf_util_set_default SPACEFISH_HOST_COLOR_SSH "green"
+
+  # ------------------------------------------------------------------------------
+  # Section
+  # ------------------------------------------------------------------------------
+
+  [ "$SPACEFISH_HOST_SHOW" = false ]; and return
+
+  if test $SPACEFISH_HOST_SHOW = 'always'; or set -q SSH_CONNECTION;
```

Comment:
Apologies for my nitpickiness 😛

In the Fish docs, [it is written](https://fishshell.com/docs/current/commands.html#test):
> When using a variable as an argument for a test operator you should almost always enclose it in double-quotes... Since it is always safe to enclose variables in double-quotes when used as `test` arguments that is the recommended practice.

I would suggest doing that in this case around both the variable and the string. 😁